### DISCUSSION TITLE
Advanced search plz ⚖️

### DISCUSSION THREAD

**Discussion by KiraiEEE** [community/community#31885] [Other Feature Feedback, Questions, & Ideas]
**hi dev goddies🍨**

thnks for da great app, i really love it and appreciate it as a person before student🍙

i hope u add this helpful option cuz without it i have to always open GitHub web app as desktop mode on ma phone when i need it so it would be very Sugoi to bring it here my senpais ✨

plz add de option to select specific programming language (or multiple) and even locale language (or multiple too)  and release date etc while searching cuz its reaaaally helpful on my studying 💖

n thx 🌸

your kouhai KiraiEEE 💫

**Comment by jfullstackdev** [comment]
you mean those features are not present in the mobile ? 

because all of that can be seen on the web UI, if you still want to use your phone, just open another browser on your phone and log in to GitHub, so you will still have those features
